packages/workflows-shared/src/binding.ts

```diff
@@ -13,13 +13,12 @@ type Env = {
 
 // this.env.WORKFLOW is WorkflowBinding
 export class WorkflowBinding extends WorkerEntrypoint<Env> implements Workflow {
-	public async create({
-		id,
-		params,
-	}: WorkflowInstanceCreateOptions): Promise<WorkflowInstance> {
-		if (!id) {
-			id = crypto.randomUUID();
-		}
+	public async create(
+		options: WorkflowInstanceCreateOptions
+	): Promise<WorkflowInstance> {
+		const id = options?.id ? options?.id : crypto.randomUUID();
+		const params = options?.params ? options?.params : {};
```

Comment:
Any reason this can't be:

```suggestion
	public async create(
		{ id = crypto.randomUUID(), params = {}}: WorkflowInstanceCreateOptions
	): Promise<WorkflowInstance> {
```